cmeutils/gsd_utils.py

```diff
@@ -32,6 +32,15 @@ def get_type_position(typename, gsd_file=None, snap=None, gsd_frame=-1):
     ]
     return typepos
 
+def unwrap_coordinates(gsd_file=None, snap=None, gsd_frame=-1):
+    snap = _validate_inputs(gsd_file, snap, gsd_frame)
+    box = snap.configuration.box
+    if box[3:].any():
+        raise ValueError("The unwrap_coordinates function only works "
+                "with orthogonal boxes."
+                )
+    pos = snap.particles.position + (snap.particles.image * box[:3])
```

Comment:
Not all gsd snapshots will have the `image` attribute.  Only those with the `dynamic=['momentum']` kwarg passed to `hoomd.dump.gsd()`. We should add some error handling for those without this. (I can't remember the value of `snap.particles.image` if momentum is not provided...)